### DISCUSSION TITLE
Steps to reproduces horitzontal tabbed/stacked

### DISCUSSION THREAD

**Discussion by Louson** [i3/i3#5040] [Q&A] [answered]
Hello, I am trying to reproduce the picture in the doc <https://i3wm.org/docs/modes.png>. I am trying to find a way using only one workspace

Let's consider forget the left part of the picture and consider only :
```
o-----------------------------o
|        Worspace 1           |
|        h-split              |
o-----------------------------o
       |                   |
       |                   |
o------------------o     o---------------------o
|        Stack     |     |        Tab          |
o------------------o     o---------------------o
    |           |            |             |
    |           |            |             |
o--------o  o--------o    o---------o    o---------o
| term 1 |  | term 2 |    | term 3  |    | term 4  |
o--------o  o--------o    o---------o    o---------o
```

I tried several combinations with the focus parent and others, but it's hard to modify the first level without modifying the top level

**Comment by Airblader** [comment, marked as answer]
This sequence will produce the desired layout (there are other ways as well, of course):

```
open (term 1)
split v
layout stacked
open (term 2)

focus parent
open (term 3)
split v
layout tabbed
open (term 4)
```

**Comment by Louson** [comment]
Hmm indeed. I thought the layer was either splitted or stacked but it seems to be something else...

**Reply by Airblader** [reply]
Layout ist a property of the parent window, so if you do `layout tabbed` you're affecting the parent of the focused window. By splitting first you introduce a new parent container to it for which you can alter the layout.
